.github/workflows/build.yaml

```diff
@@ -16,8 +16,12 @@ env:
   ROX_PRODUCT_BRANDING: RHACS_BRANDING
```

Comment:
I suggest to move this `ROX_PRODUCT_BRANDING: RHACS_BRANDING` to be only in jobs that need it but not on the global level. Additionally, I suggest to declare a matrix with only one item, `RHACS_BRANDING` to publicly shame jobs that still don't support dual branding.

Here's are ones which seem to need `ROX_PRODUCT_BRANDING: RHACS_BRANDING`:
* `build-and-push-main-rcd`
* `build-and-push-operator`

Reply:
AFAIK we never build RCD with stackrox branding
@porridge do we want operator to have double branding?

Reply:
I added task to track it https://issues.redhat.com/browse/ROX-14631

Reply:
> AFAIK we never build RCD with stackrox branding @porridge do we want operator to have double branding?

Sure, why not?

Reply:
There's https://issues.redhat.com/browse/ROX-10967 for community-branded operator.